frontend/main.js

```diff
@@ -63,6 +63,8 @@ async function startApp () {
     })
   }
 
+  sbp('okTurtles.data/set', 'LIGHTWEIGHT_CLIENT', process.env.LIGHTWEIGHT_CLIENT ?? true)
```

Comment:
@snowteamer Did you change your mind about this not being needed?

Reply:
The intent was to store a boolean, with the default being a light client. I realise now that the code as is has no clear opt out from the light client implementation.